corehq/apps/export/models/new.py

```diff
@@ -2507,7 +2507,10 @@ def _column_tuples(self):
         return section_and_product_ids
 
     def _get_product_name(self, product_id):
-        return Product.get(product_id).name
+        try:
+            return SQLProduct.objects.values_list('name', flat=True).get(product_id=product_id)
```

Comment:
Can you throw in a `domain=self.domain` here too?

Reply:
Hmm, actually I guess since it's checking product ID, it's pretty unlikely we'll get any conflicts, whatevs

Reply:
Nah. I like that idea. Especially since this may not actualy be a product id